drivers/auth/tbbr/tbbr_cot.c

```diff
@@ -6,7 +6,7 @@
 
 #include <auth_mod.h>
 #include <platform_def.h>
-#include <platform_oid.h>
+#include <tbbr_oid.h>
```

Comment:
Please move the include to be in alphabetical order?
Also, please update the copyright header to 2015-2017.